src/main/java/org/jenkinsci/plugins/ghprb/GhprbRootAction.java

```diff
@@ -102,6 +102,11 @@ public void doIndex(StaplerRequest req, StaplerResponse resp) {
                     logger.log(Level.INFO, "Skip comment on closed PR");
                     return;
                 }
+                
+                if (!issueComment.getIssue().isPullRequest()) {
```

Comment:
Will the getIssue call request data from GitHub?  If it does, and the repo requires authentication this is always going to throw an exception.  It was in the right place, it just needed to be checked once, then if it is a PR it shouldn't check again and continue through the rest of the jobs that watch that repo.